doc/classes/DisplayServer.xml

```diff
@@ -1593,6 +1607,24 @@
 				Enables or disables the given window's given [param flag]. See [enum WindowFlags] for possible values and their behavior.
 			</description>
 		</method>
+		<method name="window_set_hdr_output_enabled">
+			<return type="void" />
+			<param index="0" name="enabled" type="bool" />
+			<param index="1" name="window_id" type="int" default="0" />
+			<description>
+				Sets whether HDR output should be enabled for the window specified by [param window_id].
+				Only available on platforms that support HDR output, have HDR enabled in the system settings, and have a compatible display connected.
+			</description>
+		</method>
+		<method name="window_set_hdr_output_max_luminance">
+			<return type="void" />
+			<param index="0" name="max_luminance" type="float" />
+			<param index="1" name="window_id" type="int" default="0" />
+			<description>
+				Sets the maximum luminance of the display in nits (cd/m²) when HDR is enabled.
+				This is used to scale the HDR effect to avoid clipping.
```

Comment:
```suggestion
				Sets the maximum luminance of the display in nits (cd/m²) when HDR is enabled. If set to 0, luminance is not limited, which may look worse than setting a max luminance value suited to the display currently in use.
				This is used to scale the HDR effect to avoid clipping.
```

Reply:
I'm not so sure about this change, the max value allowed by the spec for ST2084 is 10,000 nits, which always looks blown out on any consumer display (and most of the professional ones too).  Perhaps a more reasonable default value would make more sense?

Reply:
Indeed, in real world scenarios, you'll always want luminance to be limited to a reasonable value. That said, as I understand the code, no limitation is applied if the luminance cap is set to 0 nits (the project setting defaults to that value).

That reminds me, should the default value for the HDR luminance cap be changed? The demo project uses 600 nits. We should probably see what modern games typically use as their default luminance cap value and use a value similar to that.